components/layouts/content-slider/content-slider.hbs

```diff
@@ -1,193 +1,170 @@
-            <div class="button--medium-primary content-block__button">Learn More</div>
-          </div><!-- .content-block__lower -->
-        </a><!-- .content-block--clickable -->
-
-      </div>
-      <!-- .content-slider__block -->
-
-      <div class="content-slider__block">
-
-        <a href="#" class="content-block--clickable">
-          <div class="content-block__upper">
-            <img
-              class="content-block__image"
-              src="https://via.placeholder.com/535x300"
-              role="presentation"
-              alt=""
-            />
-            <time class="content-block__date" datetime="2021-06-10">
-              <span
-                class="content-block__date-month"
-                aria-label="June"
-              >Jun</span>
-              <span class="content-block__date-day">10</span>
-            </time>
-          </div><!-- .content-block__upper -->
-
-          <div class="content-block__lower">
-            <p class="content-block__subtitle">Digitalks Webinar</p>
-            <h4 class="content-block__title">RSVP: This is a featured event that
-              is topical and relevant.</h4>
-
-            <p class="content-block__text">Lorem ipsum dolor, sit amet
-              consectetur adipisicing elit. Voluptate sit id harum aliquid ea.
-            </p>
-            <div class="button--medium-primary content-block__button">Learn More</div>
-          </div><!-- .content-block__lower -->
-        </a><!-- .content-block--clickable -->
-
-      </div>
-      <!-- .content-slider__block -->
-
-      <div class="content-slider__block">
-
-        <a href="#" class="content-block--clickable">
-          <div class="content-block__upper">
-            <img
-              class="content-block__image"
-              src="https://via.placeholder.com/535x300"
-              role="presentation"
-              alt=""
-            />
-            <time class="content-block__date" datetime="2021-06-10">
-              <span
-                class="content-block__date-month"
-                aria-label="June"
-              >Jun</span>
-              <span class="content-block__date-day">10</span>
-            </time>
-          </div><!-- .content-block__upper -->
-
-          <div class="content-block__lower">
-            <p class="content-block__subtitle">Digitalks Webinar</p>
-            <h4 class="content-block__title">RSVP: This is a featured event that
-              is topical and relevant.</h4>
-
-            <p class="content-block__text">Lorem ipsum dolor, sit amet
-              consectetur adipisicing elit. Voluptate sit id harum aliquid ea.
-            </p>
-            <div class="button--medium-primary content-block__button">Learn More</div>
-          </div><!-- .content-block__lower -->
-        </a><!-- .content-block--clickable -->
-
+  <div class="section__inner">
+    <div class="content-slider" aria-label="gallery" data-component="content-slider__content-block--clickable">
+
```

Comment:
@fienen Does that look like a reasonable naming convention for the data attribute? I welcome suggestions.

Reply:
It might be nice to qualify this name a bit better. Like, to just look at this, I'm uncertain what kind of component this is meant to be. It's not a class that exists. It's not a BEM selector, though it looks like one. I see you using it down below in index.js to instantiate the block, but I guess my question would be, why couldn't that just use the content-slider class?